workflow/rocoto/gfs_tasks.py

```diff
@@ -1781,6 +1781,40 @@ def vminmon(self):
 
         return task
 
+    def anlstat(self):
+        deps = []
+        if self.options['do_jediatmvar']:
+            dep_dict = {'type': 'task', 'name': f'{self.run}_atmanlfinal'}
+            deps.append(rocoto.add_dependency(dep_dict))
+        if self.options['do_jediocnvar']:
+            dep_dict = {'type': 'task', 'name': f'{self.run}_marineanlfinal'}
+            deps.append(rocoto.add_dependency(dep_dict))
+        if self.options['do_jedisnowda']:
+            dep_dict = {'type': 'task', 'name': f'{self.run}_snowanl'}
+            deps.append(rocoto.add_dependency(dep_dict))
+        if self.options['do_aero_anl']:
+            dep_dict = {'type': 'task', 'name': f'{self.run}_aeroanlfinal'}
+            deps.append(rocoto.add_dependency(dep_dict))
+
+        dependencies = rocoto.create_dependency(dep_condition='and', dep=deps)
+
+        resources = self.get_resource('anlstat')
+        task_name = f'{self.run}_anlstat'
+        task_dict = {'task_name': task_name,
+                     'resources': resources,
+                     'dependency': dependencies,
+                     'envars': self.envars,
+                     'cycledef': self.run.replace('enkf', ''),
```

Comment:
This job is not part of the `enkfgdas` and `enkfgfs` based on where `anlstat` is added in `gfs_cycled.py` under method `get_task_names`
```suggestion
                     'cycledef': self.run,
```